test-requirements.txt

```diff
@@ -1,7 +1,8 @@
 apache-airflow==1.10.11
 codecov==2.1.8
-marquez-python==0.6.0
+marquez-python==0.7.1
 pytest==6.0.1
 pytest-cov==2.10.1
 requests==2.24.0
+pyrfc3339==1.1
```

Comment:
possibly add a TODO to remove this line when the dependency is fixed in marquez-python

Reply:
We can remove this now with the recent `0.7.1` python client release